assets/react/controllers/Incident/ActionBox.jsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState } from "react";
+import { Button, Divider, Drawer, Popconfirm, Modal, Form } from "antd";
+import RescheduleTask from "./ActionBox/RescheduleTask";
+import ApplyPriceDiffTask from "./ActionBox/ApplyPriceDiffTask";
+import TransporterReport from "./ActionBox/TransporterReport";
+
+import { useTranslation } from "react-i18next";
+
+import store from "./incidentStore";
+
+async function _handleCancelButton(id) {
+  const httpClient = new window._auth.httpClient();
+  return await httpClient.put(
+    window.Routing.generate("api_incidents_action_item", { id }),
+    { action: "cancel_task" },
+  );
+}
+
+const styles = {
+  btn: {
+    width: "100%",
+  },
+};
+
+export default function () {
```

Comment:
Should we use `useStore` instead if importing the store directly?

Reply:
`useStore` do not auto-map state to component local bindings.
As said on a previous comment, only allow to enable reactivity.

Reply:
why not to add reactivity? So that it behaves like a standard react component